src/runner/runner.ts

```diff
@@ -0,0 +1,29 @@
+import yaml from 'js-yaml'
+import fs from 'fs'
+import { getPath, Schema } from '../migration-schema'
+
+const getConfig = async (migrationName: string): Promise<object> => {
+  return fs.readFileSync(getPath(migrationName));
+}
+
+const parseConfig = async (migrationName: string): Promise<Schema> => {
+  const config = await getConfig(migrationName);
+  return yaml.safeLoad(String(config));
+}
+
+export const run = async (migrationName: string): Promise<void> => {
+  const { up } = await parseConfig(migrationName);
+  const { table, operation, runner, query } = up;
+  
+  // run query 
```

Comment:
we can start without transactions and error handling for now. 
let's use `query` from `mysql` module and run query here. We can skip multiple runners config for now